themes/root/templates/RecordDriver/AbstractBase/export-bibtex.phtml

```diff
@@ -60,16 +60,24 @@ if (is_array($series)) {
     }
 }
 
-$author = $this->driver->tryMethod('getPrimaryAuthor');
-if (!empty($author)) {
-    echo "author = {{$author}},\n";
+$mergeAuthors = function (array $a): string {
+    return implode(
+        ' and ',
+        array_map(
+            function ($s) {
+                return str_replace(' and ', ' {and} ', $s);
+            },
+            $a
+        )
+    );
+};
+
+if ($authors = $mergeAuthors($this->driver->tryMethod('getPrimaryAuthors', [], []))) {
+    echo "author = {{$authors}},\n";
 }
 
-$secondaryAuthors = $this->driver->tryMethod('getSecondaryAuthors');
-if (is_array($secondaryAuthors)) {
-    foreach ($secondaryAuthors as $current) {
-        echo "editor = {{$current}},\n";
-    }
+if ($authors = $mergeAuthors($this->driver->tryMethod('getSecondaryAuthors', [], []))) {
```

Comment:
Would it be better to call this variable `$editors` to avoid reusing the `$authors` name? Not a big difference, but just feels slightly more clear to me.

Reply:
Changed. Not sure I like it, though, since secondary authors can be more than editors. It's just the only field we have for them.

Reply:
Agreed, it's not ideal, but it seems to be what we're stuck with.